packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -35,7 +35,7 @@
         android:label="@string/app_name"
         android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode"
         android:screenOrientation="portrait"
-        android:windowSoftInputMode="adjustResize">
```

Comment:
coolio, so this makes it similar to ios? we double checked search, filters, and edit-profile?

Reply:
Yes exactly. I check those, what do you mean by filters?